content/en/guides/create-program.md

```diff
@@ -222,11 +222,11 @@ For example, your policy could require engineers to receive approval from your o
 
 Also needed are policies to formalize and establish legal compliance procedures and to assure executive oversight for the program. (See our ebook, [Open Source Compliance in the Enterprise](https://www.linuxfoundation.org/compliance-and-security/2018/12/open-source-compliance-in-the-enterprise/), for a detailed compliance program.)
 
-You’ll also want to map out how you’ll handle the software tools that will make much of the compliance and code testing work possible by automating it and streamlining procedures for developers and contributors. For example, the [I think we need to mention SPDX and OpenChain tools from The Linux Foundation, along with help from your and working with internal supply chain teams, will internally to ensure that code brought in by third parties is also compliant.]
+You’ll also want to map out how you’ll handle the software tools that will make much of the compliance and code testing work possible by automating it and streamlining procedures for developers and contributors. We'd like to mention  along with help from internal supply chain teams (should you have them) to ensure that code brought in by third parties is also compliant.
 
-There’s a broad array of open source (e.g. FOSSology) and fee-based proprietary tools in the marketplace today that can help with these important tasks. For most open source program offices, a high-quality and comprehensive collection of tools can be brought together without the need to build your own custom tools. Again, much of this is due to open source collaboration, where contributing developers have created and refined the tools which can handle these tasks for enterprises. And if the tools don’t do exactly what your project needs, then users can contribute back to modify and improve them to gain the capabilities they seek. Getting involved in the open source community around enterprise tooling is something that your open source program office will likely support.
+There’s a broad array of open source (e.g. [FOSSology](https://www.fossology.org/), [SPDX](https://spdx.org), [OpenChain](https://www.openchainproject.org/), [ORT](https://github.com/oss-review-toolkit/ort), [Clearly Defined](https://docs.clearlydefined.io/)) as well as fee-based proprietary tools in the marketplace today that can help with these important tasks. For most open source program offices, a high-quality and comprehensive collection of tools can be brought together without the need to build your own custom tools. Again, much of this is due to open source collaboration, where contributing developers have created and refined the tools which can handle these tasks for enterprises. And if the tools don’t do exactly what your project needs, then users can contribute back to modify and improve them to gain the capabilities they seek. Getting involved in the open source community around enterprise tooling is something that your open source program office will likely support.
```

Comment:
A broad array of open source what? Please add an antecedent for that adjective.

That sentence might need some work. It doesn't flow too well.

Please don't start a sentence with And.

I guess I'm not clear on the audience here. I thought it was people looking to form an OSPO. If so, statements like that last sentence don't make sense since there isn't an OSPO to support anything yet.

Reply:
Thank you Vicki :)

All feedback should be addressed in latest push :eagle: :eye